{
  "Mayavi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300546",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291477"
  ],
  "NLopt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2254979"
  ],
  "ara": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279553"
  ],
  "blender": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291492"
  ],
  "conda-build": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300603",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291503"
  ],
  "condor": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2269253"
  ],
  "csound": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271396"
  ],
  "fedmsg": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300669",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291518"
  ],
  "jc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247039"
  ],
  "khal": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2300869",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291549"
  ],
  "krita": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2278646"
  ],
  "libprelude": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247193"
  ],
  "libpreludedb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261338"
  ],
  "petsc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250673"
  ],
  "plplot": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272975"
  ],
  "pygsl": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250675"
  ],
  "pyliblo": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2248131"
  ],
  "python-anymarkup-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261545"
  ],
  "python-avocado": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250847"
  ],
  "python-datadog": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290567"
  ],
  "python-devicely": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2277745"
  ],
  "python-devtools": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280965"
  ],
  "python-dipy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280596"
  ],
  "python-django-auth-ldap": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290343"
  ],
  "python-djvulibre": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226179"
  ],
  "python-elephant": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301142",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291685"
  ],
  "python-eventlet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290561"
  ],
  "python-flask-basicauth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261560"
  ],
  "python-gbulb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291712"
  ],
  "python-gensim": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261825"
  ],
  "python-gerritlib": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291720"
  ],
  "python-google-cloud-monitoring": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280563"
  ],
  "python-ibm-vpc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2261572"
  ],
  "python-imagehash": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226213"
  ],
  "python-ipdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280968"
  ],
  "python-jnius": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272966"
  ],
  "python-libusb1": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280555"
  ],
  "python-lmdb": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259530"
  ],
  "python-logutils": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2226233"
  ],
  "python-matplotlib-venn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280562"
  ],
  "python-memray": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280960"
  ],
  "python-mglob": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2244827"
  ],
  "python-multiprocess": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246347"
  ],
  "python-openpaperwork-core": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245800"
  ],
  "python-pathvalidate": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259547"
  ],
  "python-pdir2": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2275052"
  ],
  "python-ppft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291847"
  ],
  "python-precis_i18n": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246305"
  ],
  "python-py-gql": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246126"
  ],
  "python-pyfastnoisesimd": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2250674"
  ],
  "python-pymilter": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259637"
  ],
  "python-pytn3270": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246275"
  ],
  "python-pyunicorn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301203",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291888"
  ],
  "python-qcustomplot-pyqt": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301204",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291892"
  ],
  "python-ruffus": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259546"
  ],
  "python-scss": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245678"
  ],
  "python-setuptools_scm_git_archive": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2240107"
  ],
  "python-sphinxcontrib-openapi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279996"
  ],
  "python-stompest": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2280565"
  ],
  "python-tensordict": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301214",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291943"
  ],
  "python-testinfra": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2279554"
  ],
  "python-ufo2ft": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301217",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291961"
  ],
  "python-usort": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2271063"
  ],
  "python-webpy": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2256932"
  ],
  "python-wsaccel": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2265482"
  ],
  "python-x2go": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301218",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2291976"
  ],
  "python3-zope-fixers": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2246285"
  ],
  "qpid-dispatch": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2245601"
  ],
  "quodlibet": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259638"
  ],
  "rmlint": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2272923"
  ],
  "rpkg-util": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2276893"
  ],
  "transtats-cli": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2264228"
  ],
  "ubertooth": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301342",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2259655"
  ],
  "uwsgi": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2290556"
  ],
  "vrpn": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2247291"
  ],
  "xonsh": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2283616"
  ],
  "znc": [
    "https://bugzilla.redhat.com/show_bug.cgi?id=2301380",
    "https://bugzilla.redhat.com/show_bug.cgi?id=2292226"
  ]
}